src/test/java/bio/terra/janitor/app/common/TestUtils.java

```diff
@@ -1,30 +1,15 @@
 package bio.terra.janitor.app.common;
 
-import static bio.terra.janitor.common.JanitorObjectMapperHelper.serializeCloudResourceUid;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
 import bio.terra.generated.model.CloudResourceUid;
 import bio.terra.generated.model.GoogleProjectUid;
-import bio.terra.janitor.common.ResourceType;
-import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.google.common.collect.ImmutableMap;
-import java.sql.Timestamp;
-import java.time.Duration;
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
 import java.util.*;
-import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
-import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
 
 public class TestUtils {
```

Comment:
Do we still need this file? I think it's reasonable to move the newJsonCreateREquestBody into the api controller test, since that's where' it's being used and it's reasonable to see the request shape in that test. I don't think we need to specify defaults values for the test that we have there.
The other static final fields can move into the only test that uses them, JanitorDaoTest.